website/templates/project/settings.mako

```diff
@@ -71,28 +71,38 @@
                     <div class="panel-heading clearfix">
                         <h3 id="configureNode" class="panel-title">Configure ${node['node_type'].capitalize()}</h3>
                     </div>
-                    <div id="nodeCategorySettings" class="panel-body scripted">
-                        <h5>
-                            Category: <select data-bind="attr.disabled: disabled,
-                                                        options: categories,
-                                                        optionsValue: 'value',
-                                                        optionsText: 'label',
-                                                        value: selectedCategory"></select>
-                        </h5>
-                        <p data-bind="if: !disabled">
-                            <button data-bind="css: {disabled: !dirty()},
-                                               click: cancelUpdateCategory"
-                                    class="btn btn-default">Cancel</button>
-                            <button data-bind="css: {disabled: !dirty()},
-                                               click: updateCategory"
-                                    class="btn btn-primary">Change</button>
-                        </p>
-                        <span data-bind="css: messageClass, html: message"></span>
-
-                        <span data-bind="if: disabled" class="help-block">
-                            A top-level project's category cannot be changed.
-                        </span>
-                    </div>
+
+                    <div id="projectSettings" class="panel-body">
+                        <div class="form-group">
+                            <label>Category:</label>
+                            <select data-bind="attr.disabled: disabled,
+                                                         options: categories,
+                                                         optionsValue: 'value',
+                                                         optionsText: 'label',
+                                                         value: selectedCategory"></select>
+                            <span data-bind="if: disabled" class="help-block">
+                              A top-level project's category cannot be changed
+                            </span>
+                        </div>
+                        <div class="form-group">
+                            <label for="title">Title:</label>
```

Comment:
I can use this form to change the title in ways not allowed by the edit view on the project dashboard. Can you explore the cause while I continue with review?

1. Go to the dashboard page. Change the title to: `<img src=1 onerror='somejs'>`
2. See the error message
3. Make sure the project has a title that is not blank
4. Go to the new settings page UI for changing titles
5. Change the title to the same value tried in step 1
6. Note the lack of error message

Ideally both pages should act according to the same rules. Are the two pages using different routes to process input data?